src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -167,32 +166,32 @@ def waitForQualityGateResult(timeoutInMinutes) {
   }
 }
 
-def buildAndPushContainerImage(credentialsId, registry, imageName, tag) {
+def buildAndPushContainerImage(credentialsId, registry, projectName, tag) {
   docker.withRegistry("https://$registry", credentialsId) {
-    sh "docker-compose build --no-cache"
-    sh "docker tag $imageName $registry/$imageName:$tag"
-    sh "docker push $registry/$imageName:$tag"
+    sh "docker-compose -f docker-compose.yaml build --no-cache"
+    sh "docker tag $projectName $registry/$projectName:$tag"
+    sh "docker push $registry/$projectName:$tag"
   }
 }
 
-def deployChart(credentialsId, registry, imageName, tag, extraCommands) {
+def deployChart(credentialsId, registry, projectName, tag, extraCommands) {
   withKubeConfig([credentialsId: credentialsId]) {
-    def deploymentName = "$imageName-$tag"
+    def deploymentName = "$projectName-$tag"
     sh "kubectl get namespaces $deploymentName || kubectl create namespace $deploymentName"
-    sh "helm upgrade $deploymentName --install --namespace $deploymentName --atomic ./helm/$imageName --set image=$registry/$imageName:$tag $extraCommands"
+    sh "helm upgrade $deploymentName --install --namespace $deploymentName --atomic ./helm/$projectName --set image=$registry/$projectName:$tag $extraCommands"
   }
 }
 
-def undeployChart(credentialsId, imageName, tag) {
-  def deploymentName = "$imageName-$tag"
+def undeployChart(credentialsId, projectName, tag) {
```

Comment:
`helmChart` name would be more accurate.

Reply:
Changed to `chartName`.